index.html

```diff
@@ -3,11 +3,11 @@
   <head>
     <meta charset="utf-8">
     <title>Contentful Discovery App</title>
-    <link rel="stylesheet" href="./main.css" media="screen" charset="utf-8">
+    <link rel="stylesheet" href="/main.css" media="screen" charset="utf-8">
   </head>
   <body>
     <main></main>
   </body>
-  <script src="./main.bundle.js" charset="utf-8"></script>
+  <script src="/main.bundle.js" charset="utf-8"></script>
   <!--ANALYTICS-->
```

Comment:
the `<!--ANALYTICS-->` will be filled automatically when publishing the gh-pages and it needs that to know where to put it